app/src/androidTest/java/com/ncnf/authentication/LoginActivityTest.java

```diff
@@ -201,11 +215,11 @@ public void signUpFragmentUnsuccessfulRegisterTest(){
 
         verify(mockedAuth).register(anyString(), anyString());
 
-        onView(withId(R.id.exceptionSignUp)).check(matches(withText(containsString(unsuccessfulRegister))));
+        onView(withId(android.R.id.button1)).check(matches(isClickable()));
     }
 
     @Test
-    public void signUpFragmentUnsuccessfulRegisterTest2(){
+    public void signUpFragmentSuccessfulRegisterTest(){
         CompletableFuture<AuthenticationResponse> future = new CompletableFuture<>();
         future.complete(new AuthenticationResponse(true, null, null));
```

Comment:
Could use CompletableFuture.completedFuture(...) for convenience

Reply:
Yes I forgot this when I was coding I guess

Reply:
fixed